src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx

```diff
@@ -0,0 +1,274 @@
+/* eslint-disable jsx-a11y/no-redundant-roles */
+
+import React from 'react';
+import _ from 'lodash';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+import { isChapter33 } from '../helpers';
+
+export class StemEligibilityView extends React.Component {
+  onChange = property => {
+    this.props.onChange({
+      ...this.props.formData,
+      ...property,
+    });
+  };
+
+  iconClass = indication =>
+    classNames('fa', {
+      'fa-check': indication,
+      'vads-u-color--green': indication,
+      'fa-times': !indication,
+      'vads-u-color--gray-medium': !indication,
+    });
+
+  iconText = (indication, text) =>
+    indication ? `${text} criteria is met` : `${text} criteria is not met`;
+
+  renderIsEdithNourseRogersScholarshipCheck = () => {
+    const { isEdithNourseRogersScholarship, benefit } = this.props;
+    const check = isEdithNourseRogersScholarship && isChapter33({ benefit });
+    const question = isEdithNourseRogersScholarship && benefit === undefined;
+
+    const classes = question
+      ? 'fa fa-question vads-u-color--gray-medium'
+      : this.iconClass(check);
+
+    const text = 'Post-9/11 GI Bill beneficiary or Fry Scholarship recipient';
+    const title = question
+      ? `It is unknown if ${text} criteria is met`
+      : this.iconText(check, text);
+
+    return (
+      <li className="vads-u-margin-bottom--0">
+        <span className="fa-li">
```

Comment:
These `<span>` elements must provide text to all, or at least to screen readers, whether or not the user has met or does not meet, each criterion. Screen readers are not able to parse the icons and assign semantic meaning to them. Also need to adjust spans on lines 64 and 89.

---
![Screen Shot 2020-02-24 at 5 14 58 PM](https://user-images.githubusercontent.com/934879/75199870-db22fb80-5729-11ea-96d1-b879d9a6a0ca.png)

Reply:
Added title properties to all icons with changing text

Reply:
This approach isn't going to work across screen readers. The icons are invisible, because of the `aria-hidden="true"`. Please refactor using this format:
```html
<li class="vads-u-margin-bottom--0">
  <span class="fa-li">
    <i aria-hidden="true" class="fa fa-check vads-u-color--green"></i>
  </span>
  <!-- This span is screen reader text only -->
  <span class="vads-u-visibility--screen-reader">You answered yes to </span>
  Post-9/11 GI Bill beneficiary or Fry Scholarship recipient
</li>
```

This should also be reviewed with content for the right language around answering yes and answering no.

Reply:
changed rendering of the li elements to reflect this change